docs/installation.md

```diff
@@ -1,9 +1,11 @@
-# Clone the repository
+# Installation and Configuration for cfgov-refresh
+
+## Clone the repository
 
 Using the console, navigate to the root directory in which your projects live and clone this project's repository:
 
 ```bash
-git clone git@github.com:cfpb/cfgov-refresh.git
+git clone https://github.com/cfpb/cfgov-refresh.git
```

Comment:
When 2FA is on, `https` sucks, you have to get a new token any time it requests you to log in https://help.github.com/articles/which-remote-url-should-i-use/#cloning-with-https-urls-recommended. I still don't get why the recommend it.